odoo/addons/base/models/res_users.py

```diff
@@ -1618,7 +1618,7 @@ def _prepare_warning_for_group_inheritance(self, user):
             )
             if missing_implied_groups:
                 # prepare missing group message, by categories
-                missing_groups[group] = ", ".join(f'"{missing_group.category_id.name}: {missing_group.name}"'
+                missing_groups[group] = ", ".join(f'"{missing_group.category_id.name or _("Other")}: {missing_group.name}"'
```

Comment:
Since the specs ask to not indicate anything, shouldn't we do something like this?
```suggestion
                missing_groups[group] = ", ".join(
                    f'"{missing_group.category_id.name}: {missing_group.name}"'
                        if missing_group.category_id.name
                    else f'"{missing_group.name}"'
                    for missing_group in missing_implied_groups
                )
```

Reply:
Hello @MissingNoShiny 
After discussing with aju, she told to display other in place of nothing and warning of Other section should be displayed only in debug mode